vnext/PropertySheets/StaticLibrary/WinUI3.props

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why do we need this file now?

Reply:
ReactWindowsCore.vcxitems was setting the configuration after including React.Cpp.Props and React.Cpp.Props reads the configuration

---
In reply to: [422402558](https://github.com/microsoft/react-native-windows/pull/4840#discussion_r422402558) [](ancestors = 422402558)